ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ipv6-routing.j2

```diff
@@ -1,20 +1,15 @@
 {# eos - IPv6 Routing #}
-{% if ipv6_unicast_routing is defined and ipv6_unicast_routing == true %}
+{% if ipv6_unicast_routing is arista.avd.defined(true) %}
 !
 ipv6 unicast-routing
 {% endif %}
-{% if vrfs is defined and vrfs is not none %}
+{% if vrfs is arista.avd.defined %}
```

Comment:
As `arista.avd.defined` is included in `arista.avd.natural_sort`, if statement can be removed

Reply:
this check is redundant and should be removed